scripts/diffmazing_cropper.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+#
+# Converts diff images from Cypress into snapshots we can commit to Git.
+
+
+if [ $# -eq 0 ]
+then
+  echo ""
+  echo "Usage"
+  echo "  $0 <diff files to convert>"
+  echo ""
+  echo "Example"
+  echo "  $0 ~/downloads/*.diff.png"
+  echo ""
+  exit -1
+fi
+
+
+has_convert=$(command -v convert)
+if ! [ "$has_convert" ]
+then
+  echo ""
+  echo "Missing dependency."
+  echo "Please install ImageMagick first."
+  echo ""
+  exit -1
+fi
+
+
+
+tmp_dir=/tmp/streamlit_cropped_diffs
+### snapshots_folder=frontend/cypress/snapshots/linux/2x/
+
+diff_files=("$@")
+len=${#diff_files[@]}
+
+mkdir -p "$tmp_dir"
+rm -f "$tmp_dir"/*
```

Comment:
if we use `mktemp` above, it ensures the newly created directory has unique name, so we can drop the `mkdir` and `rm -f` commands.